Cargo.toml

```diff
@@ -57,68 +57,127 @@ members = [
 exclude = ["builtin-actors"]
 
 [workspace.package]
-authors = ["Protocol Labs"]
+authors = ["Protocol Labs", "Textile"]
 edition = "2021"
 license = "MIT OR Apache-2.0"
 license-file = "LICENSE-APACHE"
+repository = "https://github.com/recallnet/ipc"
+version = "0.1.2"
 
 [workspace.dependencies]
-ambassador = "0.3.5"
+# packages in this repo that we do NOT publish
+recall_fendermint_actor_activity_tracker = { path = "fendermint/actors/activity-tracker" }
+recall_fendermint_actor_blob_reader = { path = "fendermint/actors/blob_reader" }
+recall_fendermint_actor_blobs = { path = "fendermint/actors/blobs" }
+recall_fendermint_actor_recall_config = { path = "fendermint/actors/recall_config" }
+recall_fendermint_actors_api = { path = "fendermint/actors/api" }
+
+# packages we publish
+# NOTE: you need to change these versions as needed, i.e. they are not automatically changed by the `workspace.package.version`
+#       value above. If there is a way to avoid the need to keep this value synched in multiple places that would be better.
+recall_actor_sdk = { path = "recall/actor_sdk", version = "=0.1.2" }
```

Comment:
This may be obvious, but it's key to remember that cargo uses the semver `^` by default, so unless we explicitly use `=` or `~` the version will be updated to the highest major version available on crates.
This isn't a consideration when building or working locally, since we have a committed lock file to set versions.  crates does not capture lock files, hence we needed to be more explicit in this file.  This is especially important for any dep with a major verion of `0`, since semver allows breaking changes within a minor version change when the major version is `0`.  There are a few of these breaking changes in the `fvm_` prefixed packages.

In general I went ahead and pinned the packages we release, because we can control everything, and I've used `~` for other packages.